src/arrayReverse.test.js

```diff
@@ -1,20 +1,52 @@
-'use strict';
+"use strict";
 
 describe(`Function 'arrayReverse':`, () => {
-  const arrayReverse = require('./arrayReverse');
+  const arrayReverse = require("./arrayReverse");
 
   it(`should be declared`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
   it(`should return an array`, () => {
-
+    expect(arrayReverse(["Hello"])).toEqual(["olleH"]);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse(["", ""])).toEqual(["", ""]);
```

Comment:
The expected output for this test case is incorrect. According to the task description, the function should reverse the strings and their order. Therefore, the expected output should be `["", ""]` instead of `["", ""]`.